cmd/minikube/cmd/start.go

```diff
@@ -179,7 +179,7 @@ assumes you have already installed one of the VM drivers: virtualbox/parallels/v
 
 // runStart handles the executes the flow of "minikube start"
 func runStart(cmd *cobra.Command, args []string) {
-	console.OutStyle(console.Happy, "minikube %s on %s (%s)", version.GetVersion(), runtime.GOOS, runtime.GOARCH)
+	console.OutTemplateStyle(console.Happy, "minikube {{.version}} on {{.os}} ({{.arch}})", console.Arg{"version": version.GetVersion(), "os": runtime.GOOS, "arch": runtime.GOARCH})
```

Comment:
As we will be calling this all over the code base, can the name get any shorter? How about `OutT` and `ErrT`?

Reply:
Absolutely. Done.